wireguard-go-rs/libwg/libwg.go

```diff
@@ -23,12 +23,27 @@ import (
 )
 
 // FFI integer result codes
+// NOTE: Must be kept in sync with the Error enum in wireguard-go-rs
 const (
 	OK = C.int32_t(-iota)
+
+	// Something went wrong.
 	ERROR_GENERAL_FAILURE
+
+	// Something went wrong, but trying again might help.
 	ERROR_INTERMITTENT_FAILURE
+
+	// A bad argument was provided to libwg.
+	ERROR_INVALID_ARGUMENT
+
+	// The provided tunnel handle did not refer to an existing tunnel.
 	ERROR_UNKNOWN_TUNNEL
+
+	// The provided public key did not refer to an existing peer.
 	ERROR_UNKNOWN_PEER
+
+	// Something went wrong when enabling DAITA.
+	// TODO: consider removing this? should probably be replaced by more specific errors
 	ERROR_ENABLE_DAITA
```

Comment:
Keep or remove? Whaddya think?

Reply:
Why should we remove it? It's a possible error case. It's almost a shame we aren't more specific about the underlying error,